src/plugins/plot/legend/PlotLegendItemCollapsed.vue

```diff
@@ -116,10 +118,16 @@ export default {
     },
     beforeDestroy() {
         this.stopListening();
+        this.unsubscribeFromStalenes();
     },
     methods: {
         initialize(highlightedObject) {
             const seriesObject = highlightedObject ? highlightedObject.series : this.seriesObject;
+
+            this.unsubscribeFromStalenes = this.openmct.telemetry.subscribeToStaleness(seriesObject.domainObject, (isStale) => {
```

Comment:
typo in `unsubscribeFromStalenes` - Getting tired of the repetition were we? :)

Reply:
I must say... it was getting monotonous.

Reply:
fixed!